falco.yaml

```diff
@@ -114,6 +114,25 @@
 # disabling the automatic artifacts followed by falcoctl.
 
 
+#######################
```

Comment:
Please take a moment to deeply review this comment section and eventually fix / improve it.